.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+name: Test
+
+on:
+  push:
+    branches: ["main", "master", "develop"]
+  pull_request:
+    branches: ["main", "master", "develop"]
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: write
+
+jobs:
+  unit-test-with-pytest:
+    strategy:
+      matrix:
+        test-type: ["unit", "e2e"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Bootstrap
+        uses: ./.github/actions/bootstrap
+      - name: Exports all Github Environment Secrets
+        uses: oNaiPs/secrets-to-env-action@v1
+        with:
+          secrets: ${{ toJSON(secrets) }}
```

Comment:
I think env vars and secrets should be set in github [Secrets and variables](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions) settings instead of in the workflow files (and getting into version control).

But Github variables won't be directly available as os env var so someone has to export those. Flask app reads from os env var only.

in workflow file, I could do a bunch of
```
run: |
	export FLASK_APP= {{ var.FLASK_APP }}
...
```

but that is not DRY and brittle. Once we add any env var or secret, that workflow file needs update too.

Using the oNaiPs/secrets-to-env-action@v1 will just help me export all those secrets and vars from Gtihub to os. So flask can pick those up. 

I find this weird too (I don't remember doing this kind of thing in any previous projects) but I cannot remember what I did before lol. I always thought Github secrets and vars are directly available in os but I guess that's not the case anymore.